backend/utils/logger.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import sys
+
+
+class LoggerFactory:
+    def __init__(
+        self,
+        log_level: str | int = logging.INFO,
+        log_to_console: bool = True,
+        log_to_file: bool = False,
+        log_file_path: str = "api.log",
+        max_bytes: int = 10485760,
```

Comment:
What does these numbers mean? I know max bytes limit is 10MB but I think you should put them in fastapi's Constant class, then import back to here.

```
max_bytes: int = MAX_LOGGER_SIZE
```

Also, log_file_path and backup_count.

Reply:
These numbers are optional in case we plan to write logs to external files. I will put those value to `Constants`.